tests/run-locally.sh

```diff
@@ -9,12 +9,7 @@ trap finish EXIT
 
 docker run --detach --publish 5000:5000 --rm --name registry registry:2
 
-docker pull busybox
-docker tag busybox libero/my-dummy-project:12345678
-
-GITHUB_REF=refs/heads/master \
 DOCKER_REGISTRY=localhost:5000/ \
 DOCKER_USERNAME=foo \
 DOCKER_PASSWORD=bar \
-IMAGE_TAG=12345678 \
-github/push-image.sh my-dummy-project
+bats -r tests/
```

Comment:
Can I suggest that we use `${BATS}` (or similar) ?
... as I prefer to install npm packages locally and run with `npx bats`

Reply:
Could do with pinning too so updates don't cause tests to start to fail.

Might be better to use the [official Docker image](https://hub.docker.com/r/bats/bats) too.

Reply:
You can also install bats via a git clone and ./install.sh (see [here](https://github.com/bats-core/bats-core#installation)).

Do I understand correcly that you would prefer the use of `${BATS}` but with a default `BATS=bats` ?

Reply:
I was avoiding using the docker image as the tests need to run docker commands themselves.

I have pinned bats to 1.x.